export.md

```diff
@@ -122,11 +123,16 @@ Note: The ```Accept``` header for this request should be ```application/json```,
   - ```request``` - the full url of the original bulk data kick-off request
   - ```requiresAuthorizationToken``` - boolean value indicating whether downloading the generated files will require an authentication token. Note: This may be false in the case of signed S3 urls or an internal file server within an organization's firewall.
   - ```output``` - array of bulk data file items with one entry for each generated file. Note: If no data is returned from the kick-off request, the server should return an empty array. 
+  - ```error``` - array of error file items following the same structure as the `output` array. Note: If no errors occurred, the server should return an empty array.  Note: Only the `OperationOutcome` resource type is currently supported. 
```

Comment:
Should we point out that these should be ndjson file(s) rather than bundles of OperationOutcomes?